phone_actions.rb

```diff
@@ -0,0 +1,15 @@
+class PhoneActions
+
+  def initialize(username, password)
+    @actions_url = "https://#{username}:#{password}@api.<made_up>.com/actions/"
+  end
+
+  def call!(number, message)
+    JSON.parse(RestClient.post("@actions_url/#{text}", {message: message, number: number}))
```

Comment:
Oops, looks like `@actions_url` needs to be string interpolated. Otherwise it will literally just output the string.

It also looks like it's trying to interpolate a text variable, which wouldn't exist. It looks like this is trying to use the text API, not the call API too.

```suggestion
    JSON.parse(RestClient.post("@#{actions_url}/call", {message: message, number: number}))
```

This applies to the `@actions_url` in `text!` too, which is trying to interpolate `call`, when it should be `"#{@actions_url}/text"`.